dynamic-plugins.default.yaml

```diff
@@ -109,6 +109,27 @@ plugins:
                   if:
                     allOf:
                       - isSecurityInsightsAvailable
+  # Group: Notifications
+  - package: ./dynamic-plugins/dist/janus-idp-plugin-notifications
+    disabled: true
+    pluginConfig:
+      dynamicPlugins:
+        frontend:
+          janus-idp.backstage-plugin-notifications:
+            appIcons:
+              - name: notificationsIcon
+                module: NotificationsPlugin
+                importName: NotificationsActiveIcon
+            dynamicRoutes:
+              - path: /notifications
+                importName: NotificationsPage
+                module: NotificationsPlugin
+                menuItem:
+                  icon: notificationsIcon
+                  text: Notifications
+  - package: ./dynamic-plugins/dist/janus-idp-plugin-notifications-backend
+    disabled: true
+    pluginConfig:
```

Comment:
You don't need to provide the `pluginConfig`  field if you have no plugin-specific config to add.